hooks/pre_commit/09-version_locking.rb

```diff
@@ -0,0 +1,27 @@
+if app_option?(:lock_package_versions)
+  # evaluate version locking settings
+  cli_param = app_value(:lock_package_versions)
+  custom_config_value = get_custom_config(:lock_package_versions)
+  lock_versions = cli_param.nil? ? !!custom_config_value : cli_param # rubocop:disable Style/DoubleNegation
+
+  if lock_versions != custom_config_value
+    store_custom_config(:lock_package_versions, lock_versions)
+    kafo.config.configure_application
+  end
+
+  if packages_locked?
+    log_and_say :info, "Package versions are locked. Continuing with unlock."
+    unlock_packages
+  end
+elsif get_custom_config(:lock_package_versions)
+  # This case can happen if previously packages were locked but this is no
+  # longer supported. For example, if foreman-maintain was removed. Clearing
+  # this ensures the post hook works properly
+  store_custom_config(:lock_package_versions, nil)
+  kafo.config.configure_application
+  fail_and_exit <<~MESSAGE
+  Locking of package versions was requested but foreman-maintain couldn't be used for locking.
```

Comment:
```suggestion
  Locking of package versions was requested but foreman-maintain could not be used for locking.
```

If I'm reading it correctly, Rubocop is actually complaining about this?

Reply:
Re-ran tests and it seems not to be an issue now. Marking as resolved.